topostats/filters.py

```diff
@@ -231,20 +267,45 @@ def remove_nonlinear_polynomial(self, image: np.ndarray, mask: np.ndarray | None
 
         Parameters
         ----------
-        image: np.ndarray
-            2D numpy heightmap array of floats with a polynomial trend to remove.
-        mask: np.ndarray
-            2D numpy boolean array used to mask out any points in the image that are deemed not to be part of the
-            heightmap's background data. This argument is optional.
+        image : npt.NDArray
+            2-D numpy height-map array of floats with a polynomial trend to remove.
+        mask : npt.NDArray, optional
+            2-D Numpy boolean array used to mask any points in the image that are deemed not to be part of the
+            heightmap's background data.
```

Comment:
Really minor comment but this was changed to "height-map" above so could also do the same here to ensure consistency.

Reply:
Good spot thanks.

You can always use the "_Suggest Changes_" button to the comment box (see item 7 of [Reviewing proposed changes in a pull request - GitHub Docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request)).

Can be quicker to just make a quick edit and submit it as a suggestion.